src/clojure_tic_tac_toe/game_handler.clj

```diff
@@ -29,13 +37,41 @@
       (board/is-board-full? board)))
 
 (defn create-next-game-state
-  [{:keys [board player]} move]
+  [{:keys [board player move-strategies]} move]
   (let [updated-board (board/add-move board move player)
         next-player (switch-player board player)
         game-is-finished (is-game-finished? updated-board player)
         winner (win-checker/which-player-won? updated-board player)]
     { :board updated-board
       :player next-player
       :finished? game-is-finished
-      :winner winner }))
+      :winner winner
+      :move-strategies move-strategies }))
+
+
+(defn- get-board
+  [game-state]
+  (:board game-state))
+
+(defn display-board
```

Comment:
You've missed the point.  Try changing `io/display-board` to take a game-state instead of a board.

Reply:
`display-board` is now taking in `game-state`